components/mosart/src/riverroute/RtmIO.F90

```diff
@@ -1209,16 +1213,18 @@ subroutine ncd_io_real_var1_nf(varname, data, flag, ncid, readvar, nt)
 
     elseif (flag == 'write') then
 
+       start = 0
+       count = 0
        if (present(nt))      then
           start(1) = 1
           start(2) = nt
           count(1) = size(data)
           count(2) = 1
        else
           start(1) = 1
-          start(2) = 1
+          !start(2) = 1
```

Comment:
How about removing these entries instead of leaving them as comments?

Reply:
Ok, I will remove the commented code here and rebase the branch.